### ISSUE TITLE
[BUG] Passing Postgres password to psql terminal via set "PGPASSWORD=PASS" not working

### ISSUE THREAD

**Issue by MurzNN** [cweijan/vscode-database-client#248]
At now extension pass database password to terminal via command like this:
```
set "PGPASSWORD=superPassW0RD" && psql -U murz -h 127.0.0.1 -p 5432
```
But this isn't working, psql asks for password again:
```
set "PGPASSWORD=superPassW0RD" && psql -U murz -h 127.0.0.1 -p 5432 
Password for user murz: 
```
But this one works well:
```
PGPASSWORD=superPassW0RD psql -U murz -h 127.0.0.1 -p 5432 murz
```
and second working one:
```
export "PGPASSWORD=superPassW0RD" && psql -U murz -h 127.0.0.1 -p 5432 murz
```
Can you please update command composer to working variant? Thanks!

**Comment by cweijan** [comment]
Thanks.

**Comment by cweijan** [comment]
Fix at version 3.9.5.

**Comment by MurzNN** [comment]
Tested - now works well, thanks!
